core/src/main/java/com/confighub/core/store/Store.java

```diff
@@ -4108,15 +4109,19 @@ public List<AuditRecord> getKeyAudit( final Repository repository,
         try
         {
             StringBuilder hql = new StringBuilder();
+            Map<String, Object> userParams = new HashMap<>();
+
+            hql.append( "SELECT r FROM RevisionEntry r WHERE repositoryId = :repositoryId AND searchKey LIKE :searchKey" );
+            userParams.put( "repositoryId", repository.getId() );
+            userParams.put( "searchKey", "%|" + forKey + "|%" );
 
-            hql.append( "SELECT r FROM RevisionEntry r WHERE repositoryId = " ).append( repository.getId() ).append( " " );
-            hql.append( "AND searchKey like '%|" ).append( forKey ).append( "|%' " );
             if ( null != forUserId )
             {
-                hql.append( "AND userId = " ).append( forUserId ).append( " " );
```

Comment:
Should probably stick with the trailing-space convention for building these SQL statements

Reply:
I changed it because I think leading-space is cleaner. Each call site adds the leading space that it needs, and doesn't need to worry about whether the previous call site added it.